### ISSUE TITLE
not on web, but cvs-dev.html says use -z3 when doing checkouts

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7376]
|<img alt="ne1uno's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| ne1uno reported|
|-|-|
|Trac ID|trac#146|
|Type|enhancement|
|Created|2003-07-29 09:57:32Z|

<details><summary>Searchable metadata</summary>

```
trac-id__146 146
type__enhancement enhancement
reporter__ne1uno ne1uno
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__documentation documentation
time__1059472652000000 1059472652000000
changetime__1469316197571655 1469316197571655
version__ 
owner__acapnotic acapnotic
cc__acapnotic cc__ne1uno
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="ne1uno's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|ne1uno commented|
|-|-|

```
#!html
<pre>
howto\cvs-dev.html
Please use -z3 when doing checkouts or updates to conserve 
the limited bandwidth ...
the web page linked from download doesn't list -z3 nor in 
the the wincvs instructions

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
sheesh

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph commented|
|-|-|

```
#!html
<pre>
Can you fix this?

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)|@keturn commented|
|-|-|

```
#!html
<pre>
added -z to anoncvs command line in developers/cvs.html. 
Haven't updated the wincvs example, as I don't know where
the -z flag goes there.  Oh, and I forgot to include a "use
recent cvs or -z will hang" warning.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="ne1uno's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|ne1uno commented|
|-|-|

```
#!html
<pre>
wincvs
remote->checkout module
its on the last tab, far right on the dialog.[globals]
you have to set the option -d on the 2nd tab.[checkout 
options]
I think -z when you login isnt good so you may have to 
mention to uncheck it when logging in, not sure.
-z3 and -z6 have worked ok for me.
very confusing UI.
you have to remember also to set your directory path first 
and you stand a good chance of downloading everything again 
or grabbing all of cvs instead of just Twisted.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
docs seem up to date?

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|GitHub commented|
|-|-|

In changeset 2e8bcb3073e77f15d6c04bf601c57f653b199a2b
```
#!CommitTicketReference repository="" revision="2e8bcb3073e77f15d6c04bf601c57f653b199a2b"
Merge pull request #146 from alex/patch-2

Author: alex

Reviewer: glyph, adiroiban

Fixes: #8441

Remove use of deprecated, to-be-removed Netscape SPKI functions, and instead, determine the hash of the public key by actually extracting the public key bytes and then hashing them.  This gives the same values as the method did previously (since this is what the Netscape methods were doing).

Additionally, bump the minimum required version of pyOpenSSL to 16.0 and inform users that this is the case via the changelog.
```
